src/main/java/com/google/sps/data/EmailSender.java

```diff
@@ -14,65 +14,24 @@
 
 package com.google.sps.data;
 
-// using SendGrid's Java Library
-// https://github.com/sendgrid/sendgrid-java
-import com.sendgrid.Method;
-import com.sendgrid.Request;
 import com.sendgrid.Response;
-import com.sendgrid.SendGrid;
-import com.sendgrid.helpers.mail.Mail;
 import com.sendgrid.helpers.mail.objects.Content;
 import com.sendgrid.helpers.mail.objects.Email;
 import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.util.stream.Stream;
-import java.util.Map;
 import java.util.HashMap;
-import javax.servlet.annotation.WebServlet;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-import java.net.URL;
 
-// Handles sending emails.
-@WebServlet("/email")
-public class EmailSender {
-  // The email messages are being sent from.
-  private final Email sender;
-  private final SendGrid sg;
-
-  public EmailSender(Email sender) throws Exception {
-    this.sender = sender;
-    this.sg =
-        new SendGrid(
-            new SecretFetcher("interview-me-step-2020").getSecretValue("SENDGRID_API_KEY"));
-  }
+/**
+ * EmailSenderInterface includes the basic methods anything managing sending emails must support.
+ */
+public interface EmailSender {
 
   // Sends an email from the "from" Email to the "to" Email, with specified subject and content.
```

Comment:
Update this comment.

Reply:
Done

Reply:
This only has a "recipient" field. There shouldn't be any mention of "from" or "to".